src/asm_files.cpp

```
@@ -131,9 +131,9 @@ vector<raw_program> read_elf(const string& path, const string& desired_section,
     }
     struct stat st;
     if (stat(path.c_str(), &st)) {
-        throw std::runtime_error(string(strerror(errno)) + " opening " + path);
+        throw UnmarshalError(string(strerror(errno)) + " opening " + path);
```

cppcheck

[error] Syntax Error

(internalAstError)